### ISSUE TITLE
how is logp different from log_oldp?

### ISSUE THREAD

**Issue by danielhomola** [sid-sundrani/ppo_lightning#3]
I might be missing something subtle or hidden magic of pytorch lightning, but how is the `logp` you get in the loss function different from the `log_oldp` you pass in to calculate the `ratio` of the two?

**Comment by sidhantls** [comment]
When multiple steps of gradient descent are run on the batch, `logp` is supposed to be the log prob of the actions (in the trajectory) under each updated policy and `logp_old`  is log prob of actions under the old policy (before first update)

However, currently, multiple steps of gradient descent is not implemented, so both values are the same and `ratio` should be ~1

**Comment by danielhomola** [comment]
Thanks for the detail. I checked, and other implementations (including [openAI's](https://github.com/openai/spinningup/blob/master/spinup/algos/pytorch/ppo/ppo.py#L228) ) do the same. 

So basically this means that at the start of the first epoch the ratio is close to one and with each batch (and subsequent epoch) it's getting further and further away, so that the ratio's importance in the loss is becoming greater and will start to act as intended and described in the paper. 

Would you agree?
